{
  "checkpoints": [
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f307a25e\",\n  \"config\": {\n    \"num_layers\": 4,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 64,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.02396747676750463,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 64,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 4,\n    \"peak_lr\": 0.02396747676750463,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"1_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=64,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=4,peak_lr=0.023967,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106527.0154684,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f307a25e_1_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-06\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f307a25e_1_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-06/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407498, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407498, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f530eb30\",\n  \"config\": {\n    \"num_layers\": 3,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 32,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.0013569844635975628,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 32,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 3,\n    \"peak_lr\": 0.0013569844635975628,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"6_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=32,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=3,peak_lr=0.001357,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.4703083,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f530eb30_6_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f530eb30_6_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407545, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407545, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f532fbbe\",\n  \"config\": {\n    \"num_layers\": 4,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 32,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.022956852361325414,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 32,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 4,\n    \"peak_lr\": 0.022956852361325414,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"7_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=32,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=4,peak_lr=0.022957,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.825864,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f532fbbe_7_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f532fbbe_7_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407557, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407557, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f7cd7a02\",\n  \"config\": {\n    \"num_layers\": 1,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 32,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.00669285206617212,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 32,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 1,\n    \"peak_lr\": 0.00669285206617212,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"10_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=32,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=1,peak_lr=0.0066929,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106534.8524082,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f7cd7a02_10_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,f_2022-06-01_13-02-14\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f7cd7a02_10_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,f_2022-06-01_13-02-14/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407781, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407781, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f7cb67d0\",\n  \"config\": {\n    \"num_layers\": 5,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 64,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.00042370291449632895,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 64,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 5,\n    \"peak_lr\": 0.00042370291449632895,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"9_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=64,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=5,peak_lr=0.0004237,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106534.8384545,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f7cb67d0_9_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-14\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f7cb67d0_9_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-14/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407779, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407779, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f52eb8c4\",\n  \"config\": {\n    \"num_layers\": 2,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 64,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.0005426448695738529,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 64,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 2,\n    \"peak_lr\": 0.0005426448695738529,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"5_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=64,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=2,peak_lr=0.00054264,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.4568589,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f52eb8c4_5_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f52eb8c4_5_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407543, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407543, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f52b83d4\",\n  \"config\": {\n    \"num_layers\": 1,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 64,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.023804710102140986,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 64,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 1,\n    \"peak_lr\": 0.023804710102140986,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"4_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=64,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=1,peak_lr=0.023805,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.442321,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f52b83d4_4_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f52b83d4_4_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407541, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407541, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f528a0d8\",\n  \"config\": {\n    \"num_layers\": 5,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 64,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.04221868773610623,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 64,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 5,\n    \"peak_lr\": 0.04221868773610623,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"3_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=64,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=5,peak_lr=0.042219,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.4204931,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f528a0d8_3_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f528a0d8_3_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407539, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407539, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f5695178\",\n  \"config\": {\n    \"num_layers\": 4,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 32,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.0019546986522418317,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 32,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 4,\n    \"peak_lr\": 0.0019546986522418317,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"8_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=32,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=4,peak_lr=0.0019547,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"f5695178\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106534.8247862,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f5695178_8_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-14\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f5695178_8_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-14/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407777, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407777, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"actual_training\",\n  \"trial_id\": \"f5267ed4\",\n  \"config\": {\n    \"num_layers\": 1,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_Nhop\": 50,\n    \"hidden_dim\": 32,\n    \"warmup_iterations\": 200,\n    \"peak_lr\": 0.003648611227584545,\n    \"tot_iterations\": 281,\n    \"node_feature_dim\": 27,\n    \"edge_feature_dim\": 7,\n    \"ffn_hidden_dim\": 64,\n    \"task_dim\": 1,\n    \"ffn_dropout_rate\": 0.25,\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"end_lr\": 1e-09\n  },\n  \"local_dir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn\",\n  \"evaluated_params\": {\n    \"dataset_name\": \"9999\",\n    \"dataset_path\": \"~/projects/unified_framework/dataset/\",\n    \"edge_feature_dim\": 7,\n    \"end_lr\": 1e-09,\n    \"ffn_dropout_rate\": 0.25,\n    \"ffn_hidden_dim\": 64,\n    \"hidden_dim\": 32,\n    \"node_feature_dim\": 27,\n    \"num_kernel1_1hop\": 10,\n    \"num_kernel1_Nhop\": 10,\n    \"num_kernel2_1hop\": 20,\n    \"num_kernel2_Nhop\": 20,\n    \"num_kernel3_1hop\": 30,\n    \"num_kernel3_Nhop\": 30,\n    \"num_kernel4_1hop\": 50,\n    \"num_kernel4_Nhop\": 50,\n    \"num_layers\": 1,\n    \"peak_lr\": 0.003648611227584545,\n    \"task_dim\": 1,\n    \"tot_iterations\": 281,\n    \"warmup_iterations\": 200\n  },\n  \"experiment_tag\": \"2_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ffn_dropout_rate=0.25,ffn_hidden_dim=64,hidden_dim=32,node_feature_dim=27,num_kernel1_1hop=10,num_kernel1_Nhop=10,num_kernel2_1hop=20,num_kernel2_Nhop=20,num_kernel3_1hop=30,num_kernel3_Nhop=30,num_kernel4_1hop=50,num_kernel4_Nhop=50,num_layers=1,peak_lr=0.0036486,task_dim=1,tot_iterations=281,warmup_iterations=200\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fc5c28f5c28f5c375618c155f686561645f62756e646c655f69735f656d70747994898c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1654106530.402743,\n  \"logdir\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f5267ed4_2_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/actual_training_f5267ed4_2_dataset_name=9999,dataset_path=~_projects_unified_framework_dataset_,edge_feature_dim=7,end_lr=1e-09,ff_2022-06-01_13-02-10/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py\\\", line 901, in get_next_executor_event\\n    future_result = ray.get(ready_future)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/worker.py\\\", line 1809, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407537, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/trainable.py\\\", line 349, in train\\n    result = self.step()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 403, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 567, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train()\\u001b[39m (pid=2407537, ip=10.20.140.78, repr=actual_training)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 272, in run\\n    self._entrypoint()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 348, in entrypoint\\n    return self._trainable_func(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/function_runner.py\\\", line 640, in _trainable_func\\n    output = fn()\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 390, in _inner\\n    inner(config, checkpoint_dir=None)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/ray/tune/utils/trainable.py\\\", line 381, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/hypertuning.py\\\", line 267, in actual_training\\n    trainer.fit(model=model, datamodule=data_module)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 740, in fit\\n    self._call_and_handle_interrupt(\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 685, in _call_and_handle_interrupt\\n    return trainer_fn(*args, **kwargs)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 777, in _fit_impl\\n    self._run(model, ckpt_path=ckpt_path)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1138, in _run\\n    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 1438, in _call_setup_hook\\n    self.datamodule.setup(stage=fn)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/pytorch_lightning/core/datamodule.py\\\", line 474, in wrapped_fn\\n    fn(*args, **kwargs)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/data.py\\\", line 147, in setup\\n    split_idx = self.dataset['dataset'].get_idx_split(seed=self.seed)\\n  File \\\"/home/live-lab/projects/unified_framework/kgnn/wrapper.py\\\", line 640, in get_idx_split\\n    split_dict = torch.load(f'data_split/{self.dataset}_seed{seed}.pt')\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 699, in load\\n    with _open_file_like(f, 'rb') as opened_file:\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 231, in _open_file_like\\n    return _open_file(name_or_buffer, mode)\\n  File \\\"/home/live-lab/environments/venv/ray-env/lib/python3.9/site-packages/torch/serialization.py\\\", line 212, in __init__\\n    super(_open_file, self).__init__(open(name, mode))\\nFileNotFoundError: [Errno 2] No such file or directory: 'data_split/9999_seed42.pt'\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059583010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d948c056f72646572944b0075628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9468134b0075628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f948c0a5f6375725f6f72646572944b0075622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}"
  ],
  "runner_data": {
    "_insufficient_resources_manager": {
      "_type": "CLOUDPICKLE_FALLBACK",
      "value": "8005958b000000000000008c277261792e74756e652e696e73756666696369656e745f7265736f75726365735f6d616e61676572948c1c496e73756666696369656e745265736f75726365734d616e616765729493942981947d94288c185f6e6f5f72756e6e696e675f747269616c735f73696e6365944affffffff8c0f5f6c6173745f747269616c5f6e756d944affffffff75622e"
    },
    "_max_pending_trials": 1,
    "_metric": "logAUC",
    "_total_time": 0,
    "_iteration": 20,
    "_has_errored": true,
    "_fail_fast": false,
    "_server_port": null,
    "_cached_trial_decisions": {},
    "_queued_trial_decisions": {},
    "_updated_queue": true,
    "_should_stop_experiment": false,
    "_local_checkpoint_dir": "/home/live-lab/projects/unified_framework/kgnn/tune_kgnn",
    "_remote_checkpoint_dir": null,
    "_stopper": {
      "_type": "CLOUDPICKLE_FALLBACK",
      "value": "80059527000000000000008c107261792e74756e652e73746f70706572948c0b4e6f6f7053746f707065729493942981942e"
    },
    "_resumed": false,
    "_start_time": 1654106526.832029,
    "_last_checkpoint_time": -Infinity,
    "_session_str": "2022-06-01_13-02-06",
    "checkpoint_file": "/home/live-lab/projects/unified_framework/kgnn/tune_kgnn/experiment_state-2022-06-01_13-02-06.json",
    "_checkpoint_period": "auto",
    "launch_web_server": false
  },
  "stats": {
    "start_time": 1654106526.832029,
    "timestamp": 1654106538.4573302
  }
}